src/elasticsearch/elasticsearch-sync.service.ts

```
@@ -0,0 +1,766 @@
+          const fullUserDoc = await applicationDataProvider(userId, cohortId);
+          if (fullUserDoc) {
+            // Convert IApplication to IUser format for Elasticsearch update
+            const userDocument: IUser = {
+              userId,
+              profile: await this.buildUserProfile(userId),
+              applications: [fullUserDoc],
+              courses: [],
+              createdAt: new Date().toISOString(),
+              updatedAt: new Date().toISOString(),
+            };
+            
```

ESLint

[error] Delete `············`

(prettier/prettier)